test/chat-plugins/trivia.js

```diff
@@ -1,58 +1,44 @@
 'use strict';
 
 const assert = require('assert');
-const Module = require('module');
 
 const userUtils = require('../../dev-tools/users-utils');
 const User = userUtils.User;
 const Connection = userUtils.Connection;
 
-// We can't import trivia outside of the test suite's context, since the trivia
-// module doesn't have access to any of the globals defined in app.js from this
-// context. For now we'll just construct a skeleton module representing the
-// trivia module and wait...
-const triviaModule = (() => {
-	let pathname = require.resolve('../../chat-plugins/trivia');
-	let ret = new Module(pathname, module);
-	Module._preloadModules(ret);
-
-	return ret;
-})();
-
 let SCORE_CAPS;
 let Trivia;
 let FirstModeTrivia;
 let TimerModeTrivia;
 let NumberModeTrivia;
 
-function makeUser(name, connection) {
+const makeUser = (name, connection) => {
```

Comment:
Didn't I just explain...? I guess you could call it a holdover from writing other languages where constants and immutable variables are encouraged/mandatory